federationtypes.go

```diff
@@ -248,17 +248,21 @@ func (r RespState) Events() ([]Event, error) {
 
 // Check that a response to /state is valid.
 func (r RespState) Check(ctx context.Context, keyRing JSONVerifier) error {
+	return checkStateAndAuthChain(ctx, r.StateEvents, r.AuthEvents, keyRing)
+}
+
+func checkStateAndAuthChain(ctx context.Context, StateEvents []Event, AuthEvents []Event, keyRing JSONVerifier) error {
```

Comment:
`StateEvents` and `AuthEvents` should be in `lowerCamelCase`.

You may also consider adding a docstring for this function.